sklearn/tree/export.py

```diff
@@ -61,7 +61,11 @@ def _color_brew(n):
     return color_list
 
 
-def export_graphviz(decision_tree, out_file="tree.dot", max_depth=None,
+class Sentinel:
+    __repr__ = lambda x: '"tree.dot"'
```

Comment:
i think you are commenting on an outdated diff? the linter picked this up and i fixed it in b69d5fa